modules/user-guide/pages/setting-up-the-bitbucket-application-link.adoc

```diff
@@ -0,0 +1,94 @@
+:_content-type: PROCEDURE
+:description: Setting up the Bitbucket application link
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, private-bitbucket, private, bitbucket, bitbucket-repo, bitbucket-repository
+:navtitle: Setting up the Bitbucket application link
+// :page-aliases:
+
+[id="setting-up-the-bitbucket-application-link_{context}"]
+= Setting up the Bitbucket application link
+
+To be able to access a private Git repository that is hosted using Bitbucket, set up a Bitbucket application link as follows:
+
+.Prerequisites
+
+* You are logged in to Bitbucket.
+* link:https://www.openssl.org/[`openssl`] is installed in the operating system you are using.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. On a command line, generate the *privatepkcs8-stripped.pem* key in the required format:
++
+----
+$ openssl genrsa -out private.pem 2048
+
+$ openssl pkcs8 -topk8 -inform pem -outform pem -nocrypt -in private.pem -out privatepkcs8.pem
+
+$ cat privatepkcs8.pem | sed 's/-----BEGIN PRIVATE KEY-----//g' |  sed 's/-----END PRIVATE KEY-----//g' | tr -d '\n' > privatepkcs8-stripped.pem
+----
+
+. On a command line, generate the *public-stripped.pub* key in the required format:
++
+----
+$ openssl rsa -in private.pem -pubout > public.pub
+
+$ cat public.pub | sed 's/-----BEGIN PUBLIC KEY-----//g' | sed 's/-----END PUBLIC KEY-----//g' | tr -d '\n' > public-stripped.pub
+
+----
+
+. On a command line, generate a consumer key:
++
+----
+$ openssl rand -base64 24 > bitbucket-consumer-key
+----
+
+. On a command line, generate a shared secret:
++
+----
+$ openssl rand -base64 24 > bitbucket-shared-secret
+----
+
+. Go to *Administration* *>* *Application Links*
+//I can't find *Administration* when I am logged in to Bitbucket. max-cx
+//using https://youtu.be/djXwI9OGvhQ?t=117 as a reference (also asking the SME). max-cx
+//in the above YouTube tutorial: `<host>/bitbucket/plugins/servlet/applinks/listApplicationLinks`
+
+. Enter `{prod-url}/` into the URL field and click *Create new link*.
+
+. Under *The supplied Application URL has redirected once*, check the *Use this URL* checkbox and click *Continue*.
+
+. Enter *{prod-short}* as the *Application Name*.
+
+. Select *Generic Application* as the *Application Type*.
+
+. Enter *{prod-short}* as the *Service Provider Name*.
+
+. Paste the content of the *bitbucket-consumer-key* file as the *Consumer key*.
+
+. Paste the content of the *bitbucket-shared-secret* file as the *Shared secret*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/request-token` as the *Request Token URL*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/access-token` as the *Access token URL*.
+
+. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/access-token` as the *Authorize URL*.
```

Comment:
```suggestion
. Enter `__<bitbucket_server_url>__/plugins/servlet/oauth/authorize` as the *Authorize URL*.
```
Same suggestion/question as I wrote here: https://github.com/eclipse-che/blog/pull/24#discussion_r837958784

Reply:
Thank you. Fixed.